third_party/xla_client/pjrt_computation_client.cc

```diff
@@ -71,9 +69,11 @@ PjRtComputationClient::PjRtComputationClient() {
     bool async = sys_util::GetEnvBool(env::kEnvPjrtAsyncGpuClient, true);
     /* TODO(jonbolin): Set allowed_devices based on local ordinal */
     auto allowed_devices = std::make_optional<std::set<int>>(std::set{0});
-    client_ = xla::GetGpuClient(/*asynchronous=*/async, GpuAllocatorConfig{},
-                           /*distributed_client=*/nullptr, /*node_id=*/0,
-                           allowed_devices=allowed_devices).ValueOrDie();
+    client_ = xla::GetStreamExecutorGpuClient(
```

Comment:
@will-cromar I guess this is fine since TFRT does not support GPU yet?

Reply:
Yes, that's correct